Dockerfile

```diff
@@ -12,9 +176,17 @@ RUN ln -s /usr/bin/unzip /l/local/bin/unzip
 RUN ln -s /usr/bin/convert /l/local/bin/convert
 RUN ln -s /usr/local/bin/kdu_expand /l/local/bin/kdu_expand
 RUN ln -s /usr/local/bin/kdu_compress /l/local/bin/kdu_compress
+RUN ln -s /usr/bin/plackup /l/local/bin/plackup
 RUN /bin/bash -c 'for cmd in pamflip jpegtopnm tifftopnm bmptopnm pngtopam ppmmake pamcomp pnmscalefixed pamscale pnmrotate pnmpad pamtotiff pnmtotiff pnmtojpeg pamrgbatopng ppmtopgm pnmtopng; do ln -s /usr/bin/$cmd /l/local/bin; done'
 
 WORKDIR /htapps/babel/imgsrv
+
+RUN mkdir /htapps/babel/cache
+RUN chmod 4777 /htapps/babel/cache
+
+RUN mkdir /htapps/babel/logs
+RUN chmod 4777 /htapps/babel/logs
+
 RUN ln -s /htapps/babel /htapps/test.babel
 RUN cd /htapps/babel; git clone https://github.com/hathitrust/mdp-web.git
```

Comment:
I wonder if it would make sense to treat `mdp-web` and `imgsrv-sample-data` the same, i.e. they should be checked-out repositories at the same level of imgsrv rather than the `Dockerfile` having the reponsibility for cloning? Certainly if we want to co-develop these things we wouldn't want to bake into the image...